### PR TITLE
[DOCS] Update README_zh_CN.md

### PR DESCRIPTION
**Pull request by zirnc**
**Describe the pull request**

- #### What does your PR fix?
  cmake -> CMake

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  <all / linux, windows, ...>, <Yes/No>

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  `Your answer`

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  <Yes / I am still working on this PR>

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: README_zh_CN.md [hunk 1]
```diff
@@ -5,9 +5,9 @@
 Vcpkg 可帮助您在 Windows、 Linux 和 MacOS 上管理 C 和 C++ 库。
 这个工具和生态链正在不断发展，我们一直期待您的贡献！
 
-若您从未使用过vcpkg，或者您正在尝试了解如何使用vcpkg，请查阅 [入门](#入门) 章节。
+若您从未使用过 vcpkg，或者您正在尝试了解如何使用 vcpkg，请查阅 [入门](#入门) 章节。
```
Comment (LilyWangLL) [inline README_zh_CN.md:8]: I don't think adding space is needed. @JackBoosY

Comment (JackBoosY) [reply README_zh_CN.md:8]: Yes, it's not needed.

Comment (ChungZH) [reply README_zh_CN.md:8]: > I don't think adding space is needed. @JackBoosY

https://github.com/sparanoid/chinese-copywriting-guidelines

Adding spaces makes it prettier and easier to read.

Comment (ChungZH) [reply README_zh_CN.md:8]: image

**Review by Cheney-W — APPROVED** [review]

File: README_zh_CN.md (hunk 1 above)
Comment (LilyWangLL) [reply README_zh_CN.md:8]: OK, I agree with this change for this reason. Thanks @ChungZH's work~

**Review by LilyWangLL — APPROVED** [review]

**Review by Adela0814 — APPROVED** [review]

**Review by JackBoosY — APPROVED** [review]

**Comment by JavierMatosD** [comment]
Thank you!

**Pull request merged**